jinahub/encoders/audio/VGGISHAudioEncoder/vggish_audio_encoder.py

```diff
@@ -33,6 +36,19 @@ def __init__(self,
         self.pca_path = pca_path
         self.default_traversal_paths = default_traversal_paths or ['r']
 
+        Path(os.path.join(cur_dir, 'models')).mkdir(exist_ok=True)
+
+        if not os.path.exists(self.model_path):
+            r = requests_.get('https://storage.googleapis.com/audioset/vggish_model.ckpt')
+            with open(model_path, 'wb') as f:
+                f.write(r.content)
+
+        if not os.path.exists(self.pca_path):
+            r = requests_.get('https://storage.googleapis.com/audioset/vggish_pca_params.npz')
+            with open(pca_path, 'wb') as f:
+                f.write(r.content)
+
```

Comment:
need to refactor executor's parameters. We should use the same folder (default is `.models`) for vggish and pca model. 
```
self.model_path = Path(model_path)
self.vgg_model_path = self.model_path / 'vggish_model.ckpt'
self.pca_model_path = self.model_path / 'vggish_pca_params.npz'
```

Reply:
Addressed, please see [here](https://github.com/jina-ai/executors/pull/31/commits/c8d0239634a96d11d2f0b684c83a65ab6b5f78d0) for the change. Also added error checking when issuing requests to download models as discussed, thank you.